homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -299,3 +321,75 @@ def _async_update_shade_from_group(self):
             return
         self._async_process_new_shade_data(self.coordinator.data[self._shade.id])
         self.async_write_ha_state()
+
+
+class SilhouettePowerViewShade(PowerViewShade):
+    """Class to represent a Silhouette (Type 23) blind."""
+
+    def __init__(self, coordinator, device_info, room_name, shade, name):
+        """Initialize the shade."""
+        super().__init__(coordinator, device_info, room_name, shade, name)
+        self._max_tilt = 32767
+        self._attr_current_cover_tilt_position = 0
+        self._attr_supported_features |= (
+            CoverEntityFeature.OPEN_TILT
+            | CoverEntityFeature.CLOSE_TILT
+            | CoverEntityFeature.STOP_TILT
+            | CoverEntityFeature.SET_TILT_POSITION
+        )
+
+    async def async_open_cover_tilt(self, **kwargs):
+        """Open the cover tilt."""
+        self._async_update_from_command(await self._shade.tilt_open())
+
+    async def async_close_cover_tilt(self, **kwargs):
+        """Close the cover tilt."""
+        self._async_update_from_command(await self._shade.tilt_close())
+
+    async def async_set_cover_tilt_position(self, **kwargs):
+        """Move the cover tilt to a specific position."""
+        if ATTR_TILT_POSITION not in kwargs:
+            return
```

Comment:
This shouldn't be possible as its required by the schema
```
    component.async_register_entity_service(
        SERVICE_SET_COVER_TILT_POSITION,
        {
            vol.Required(ATTR_TILT_POSITION): vol.All(
                vol.Coerce(int), vol.Range(min=0, max=100)
            )
        },
        "async_set_cover_tilt_position",
        [CoverEntityFeature.SET_TILT_POSITION],
    )
```

Reply:
Sorry youve lost me, whats wrong here?

Reply:
We can drop the guard as its not possible for `ATTR_TILT_POSITION` to not be in kwargs since the service call requires it.

Reply:
Gotcha, "copied" this from the set position method which also has a guard

Reply:
Yup that one is unreachable as well since the set_cover_position service will never call it without it... but thats an existing problem to fix for the future in a separate PR
```
    component.async_register_entity_service(
        SERVICE_SET_COVER_POSITION,
        {
            vol.Required(ATTR_POSITION): vol.All(
                vol.Coerce(int), vol.Range(min=0, max=100)
            )
        },
        "async_set_cover_position",
        [CoverEntityFeature.SET_POSITION],
```